editor/liveblocks.config.ts

```diff
@@ -86,6 +86,36 @@ export type ThreadMetadata = {
   sceneId?: string
   remixLocationRoute?: string
   resolved: boolean
+  userReadStatuses: string
+}
+
+export type UserReadStatuses = { [userId: string]: boolean }
+
+export function isUserReadStatuses(statuses: any): statuses is UserReadStatuses {
+  return (
+    statuses != null &&
+    typeof statuses === 'object' &&
+    Object.keys(statuses).every((v) => typeof v === 'string') &&
+    Object.values(statuses).every((v) => typeof v === 'boolean')
+  )
+}
+
+export function parseUserReadStatuses(userReadStatuses: string): UserReadStatuses {
+  try {
```

Comment:
We have a simple parser infrastructure in `value-parser-utils.ts` that you could use in this which does all the heavy lifting.